install/kubernetes/cilium/templates/cilium-ingress-drone/daemonset.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.ingressController.enabled -}}
+{{- if ne .Values.cni.chainingMode "none" -}}
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: cilium-ingress-drone
+  namespace: {{ .Release.Namespace }}
+  labels:
+    io.cilium/app: ingress-drone
+    name: cilium-ingress-drone
+spec:
+  selector:
+    matchLabels:
+      io.cilium/app: ingress-drone
+      'reserved:ingress': ''
```

Comment:
does this actually work for policy? :fearful:

Reply:
^ I don't think so, I tried something similar locally and got:

```
The Pod "xwing" is invalid: metadata.labels: Invalid value: "reserved:ingress": name part must consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyName',  or 'my.name',  or '123-abc', regex used for validation is '([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]')
```

Reply:
I'm not quite sure the best way to handle this, but it may be something like just extending the policy selector for the ingress entity to select the label `io.cilium/app: ingress-drone`.